src/scripts/boot.lua

```diff
@@ -550,8 +550,8 @@ function love.init()
 			usedpiscale = c.window.usedpiscale,
 			x = c.window.x,
 			y = c.window.y,
+			title = c.window.title or c.title
```

Comment:
I would use `title = c.window.title or "Untitled"` here, to be consistent with previous versions.

Reply:
I was simply moving the line 554 to here, but looking the documentation there doesn't appear to be a config option for an application title, only the window. Will change as suggested.